internal/mode/static/nginx/config/servers.go

```diff
@@ -23,82 +23,33 @@ const (
 	rootPath             = "/"
 )
 
-// httpBaseHeaders contains the constant headers set in each HTTP server block.
-var httpBaseHeaders = []http.Header{
-	{
-		Name:  "Host",
-		Value: "$gw_api_compliant_host",
-	},
-	{
-		Name:  "X-Forwarded-For",
-		Value: "$proxy_add_x_forwarded_for",
-	},
-	{
-		Name:  "Upgrade",
-		Value: "$http_upgrade",
-	},
-	{
-		Name:  "Connection",
-		Value: "$connection_upgrade",
-	},
-	{
-		Name:  "X-Real-IP",
-		Value: "$remote_addr",
-	},
-	{
-		Name:  "X-Forwarded-Proto",
-		Value: "$scheme",
-	},
-	{
-		Name:  "X-Forwarded-Host",
-		Value: "$host",
-	},
-	{
-		Name:  "X-Forwarded-Port",
-		Value: "$server_port",
-	},
-}
-
-// grpcBaseHeaders contains the constant headers set in each gRPC server block.
-var grpcBaseHeaders = []http.Header{
-	{
-		Name:  "Host",
-		Value: "$gw_api_compliant_host",
-	},
-	{
-		Name:  "X-Forwarded-For",
-		Value: "$proxy_add_x_forwarded_for",
-	},
-	{
-		Name:  "Authority",
-		Value: "$gw_api_compliant_host",
-	},
-	{
-		Name:  "X-Real-IP",
-		Value: "$remote_addr",
-	},
-	{
-		Name:  "X-Forwarded-Proto",
-		Value: "$scheme",
-	},
-	{
-		Name:  "X-Forwarded-Host",
-		Value: "$host",
-	},
-	{
-		Name:  "X-Forwarded-Port",
-		Value: "$server_port",
-	},
-}
-
-func (g GeneratorImpl) newExecuteServersFunc(generator policies.Generator) executeFunc {
+var authorityHeader = http.Header{
+	Name:  "Authority",
+	Value: "$gw_api_compliant_host",
+}
+
+var connectionHeader = http.Header{
+	Name:  "Connection",
+	Value: "$connection_upgrade",
+}
+
+var upgradeHeader = http.Header{
+	Name:  "Upgrade",
+	Value: "$http_upgrade",
+}
+
+func (g GeneratorImpl) newExecuteServersFunc(generator policies.Generator, um UpstreamMap) executeFunc {
```

Comment:
Let's use `upstreamMap` as the var name throughout, instead of `um`.

Reply:
Removed upstream map so this comment is no longer applicable.